tests/ppx/react/expected/v4.res.txt

```diff
@@ -16,3 +16,55 @@ module AnotherName = {
     \"V4$AnotherName$anotherName"
   }
 }
+
+module Rec = {
+  type props = {}
+
+  let rec make = {
+    @merlin.focus
+    let \"make$Internal" = (_: props) => {
+      make(({}: props))
+    }
+    let make = {
+      let \"V4$Rec" = props => \"make$Internal"(props)
```

Comment:
This part seems to be doing noting in the case of recursive functions.
That was true already before, so that's OK. Just, not very useful.

Reply:
Since recursive components are rare, let's just move on.

Reply:
This part seems to be making this expression in js output:
```js
function make(_props) {
  while(true) {
    _props = {};
    continue ;
  };
}

var V4$Rec = { // <- here
  make: make
};
```